common/src/main/resources/assets/now-playing/lang/zh_tw.json

```diff
@@ -0,0 +1,96 @@
+  "music.${mod_id}.taswell": "C418 - Taswell",
+  "music.${mod_id}.subwoofer_lullaby": "C418 - Subwoofer Lullaby",
+  "music.${mod_id}.living_mice": "C418 - Living Mice",
+  "music.${mod_id}.haggstrom": "C418 - Haggstrom",
+  "music.${mod_id}.danny": "C418 - Danny",
+  "music.${mod_id}.key": "C418 - Key",
+  "music.${mod_id}.oxygene": "C418 - Oxygène",
+  "music.${mod_id}.dry_hands": "C418 - Dry Hands",
+  "music.${mod_id}.wet_hands": "C418 - Wet Hands",
+  "music.${mod_id}.mice_on_venus": "C418 - Mice on Venus",
+  "music.${mod_id}.aerie": "Lena Raine - Aerie",
+  "music.${mod_id}.ancestry": "Lena Raine - Ancestry",
+  "music.${mod_id}.a_familiar_room": "Aaron Cherof - A Familiar Room",
+  "music.${mod_id}.an_ordinary_day": "Kumi Tanioka - An Ordinary Day",
+  "music.${mod_id}.bromeliad": "Aaron Cherof - Bromeliad",
+  "music.${mod_id}.comforting_memories": "Kumi Tanioka - Comforting Memories",
+  "music.${mod_id}.crescent_dunes": "Aaron Cherof - Crescent Dunes",
+  "music.${mod_id}.deeper": "Lena Raine - Deeper",
+  "music.${mod_id}.echo_in_the_wind": "Aaron Cherof - Echo in the Wind",
+  "music.${mod_id}.eld_unknown": "Lena Raine - Eld Unknown",
+  "music.${mod_id}.endless": "Lena Raine - Endless",
+  "music.${mod_id}.featherfall": "Aaron Cherof - Featherfall",
+  "music.${mod_id}.firebugs": "Lena Raine - Firebugs",
+  "music.${mod_id}.floating_dream": "Kumi Tanioka - Floating Dream",
+  "music.${mod_id}.infinite_amethyst": "Lena Raine - Infinite Amethyst",
+  "music.${mod_id}.komorebi": "Kumi Tanioka - komorebi",
+  "music.${mod_id}.labyrinthine": "Lena Raine - Labyrinthine",
+  "music.${mod_id}.left_to_bloom": "Lena Raine - Left to Bloom",
+  "music.${mod_id}.one_more_day": "Lena Raine - One More Day",
+  "music.${mod_id}.pokopoko": "Kumi Tanioka - pokopoko",
+  "music.${mod_id}.puzzlebox": "Aaron Cherof - Puzzlebox",
+  "music.${mod_id}.stand_tall": "Lena Raine - Stand Tall",
+  "music.${mod_id}.watcher": "Aaron Cherof - Watcher",
+  "music.${mod_id}.wending": "Lena Raine - Wending",
+  "music.${mod_id}.yakusoku": "Kumi Tanioka - yakusoku",
+  "music.${mod_id}.axolotl": "C418 - Axolotl",
+  "music.${mod_id}.dragon_fish": "C418 - Dragon Fish",
+  "music.${mod_id}.shuniji": "C418 - Shuniji",
+  "music.${mod_id}.concrete_halls": "C418 - Concrete Halls",
+  "music.${mod_id}.dead_voxel": "C418 - Dead Voxel",
+  "music.${mod_id}.warmth": "C418 - Warmth",
+  "music.${mod_id}.ballad_of_the_cats": "C418 - Ballad of the Cats",
+  "music.${mod_id}.chrysopoeia": "Lena Raine - Chrysopoeia",
+  "music.${mod_id}.rubedo": "Lena Raine - Rubedo",
+  "music.${mod_id}.so_below": "Lena Raine - So Below",
+  "music.${mod_id}.boss": "C418 - Boss",
+  "music.${mod_id}.the_end": "C418 - The End",
+  "music.${mod_id}.mutation": "C418 - Mutation",
+  "music.${mod_id}.moog_city_2": "C418 - Moog City 2",
+  "music.${mod_id}.beginning_2": "C418 - Beginning 2",
+  "music.${mod_id}.floating_trees": "C418 - Floating Trees",
+  "music.${mod_id}.alpha": "C418 - Alpha"
```

Comment:
You don't need to include the song titles, they'll automatically fall back to english if they're not included here.

Reply:
Okay, Should I remove the song title strings?